routes/login/logout.py

```diff
@@ -0,0 +1,21 @@
+from flask import render_template, Blueprint, redirect, url_for, flash
+from flask_login import logout_user, login_required, current_user
+from globals.globals import login_manager
+from db.models import User
+
+
+logout_blueprint = Blueprint('logout', __name__)
+
+
+@logout_blueprint.route("/logout")
+def logout():
+    if is_active():
+        logout_user()
+        flash('Logout successful.')
+        return redirect(url_for('login.login')) 
+    else:
```

Comment:
there's no need to have the else here. you can just write the if and leave the rest outside. E.G:

```
if foo():
    bar

baz
```